tests/orm/data/code/test_portable.py

```diff
@@ -96,3 +96,20 @@ def test_get_execname(tmp_path):
     code = PortableCode(label='some-label', filepath_executable='bash', filepath_files=tmp_path)
     with pytest.warns(AiidaDeprecationWarning):
         assert code.get_execname() == 'bash'
+
+
+def test_portablecode_repo_dump(tmp_path, chdir_tmp_path):
+    """Test that the node repository contents of an orm.PortableCode are dumped upon YAML export."""
+    filepath_files = tmp_path / 'tmp'
+    filepath_files.mkdir()
+    (filepath_files / 'bash').touch()
+    (filepath_files / 'subdir').mkdir()
+    (filepath_files / 'subdir/test').touch()
+    code = PortableCode(label='some-label', filepath_executable='bash', filepath_files=filepath_files)
+    code.store()
+    chdir_tmp_path
```

Comment:
no-op? Are you under the impression that the `chdir_tmnp_path` fixture only actually changes dir when evaluated? I am pretty sure it does it automatically at the start of the test. So don't think this does anything

Reply:
Yes, you're right. Wasn't sure if the fixtures would be automatically executed. Now I know :)